### ISSUE TITLE
500 error on repo creation

### ISSUE THREAD

**Issue by Nohj101** [microsoft/powerapps-tools#863]
Testing project creation from the ALM Accelerator for Makers after setting up the COE Starter Kit, and we have an issue on the create repo step. It gives a 500 error, and gives a general error of "Cannot create repo with name ****"

We are using a non-paid git hub org account setup.

{
"statusCode": 500,
"headers": {
"x-ms-workflow-run-id": "08585855019469477994336743357CU53",
"x-ms-correlation-id": "d7190980-78c4-4871-9963-2844111e9f86",
"x-ms-client-tracking-id": "08585855020515451346868756993CU78",
"x-ms-trigger-history-name": "08585855019469477994336743357CU53",
"x-ms-execution-location": "westus",
"x-ms-workflow-system-id": "/locations/westus/scaleunits/prod-109/workflows/2e83030f9f664db6bba1fe7800c25f65",
"x-ms-workflow-id": "2e83030f9f664db6bba1fe7800c25f65",
"x-ms-workflow-version": "08585855072469115110",
"x-ms-workflow-name": "f3e8da8e-f407-a2fa-1662-c5187c99fbe1",
"x-ms-tracking-id": "1f2a83f7-4a2c-4412-b417-4eaef40d0d9f",
"Content-Type": "text/plain; charset=utf-8"
},
"body": "Error Creating Repo with name thistime"
}

wondering if this may have something to do with the admin account we are using for creation.  It is a service account without an email address, so that account doesn't have access to the repo.

![image](https://user-images.githubusercontent.com/60517236/111996931-83032800-8ae8-11eb-8d4e-cc9b14f576c6.png)

**Comment by Nohj101** [comment]
Since it is a 500 error, not sure the admin account is the issue.

**Comment by mikefactorial** [comment]
@Nohj101 can you verify if the user you used to create the GitHub connection has permissions to create new repositories under your org? 
![image](https://user-images.githubusercontent.com/42348035/112001284-33772900-8af5-11eb-9f14-199330cdbc0b.png)

**Comment by Nohj101** [comment]
Yes, the account used there can create repos for the organization.  I tested it now just to be sure.

**Comment by zafarabhatti** [comment]
Hi All, Has somebody got the solution for it? **I have run into the same problem while using AML Accelerator Admin**. It can create the environment instance but then flow fails on repository creation in Github.

<img width="408" alt="AML Error" src="https://user-images.githubusercontent.com/86948940/125310529-f8fafb80-e332-11eb-9548-1a19dcbd620c.PNG">
